types/qtypings-browser-global/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts"
```

Comment:
Include test file as well

Reply:
I am releasing a package that solely defines some global types, such as QJson, for convenient use in projects. There is no specific implementation, and no test files are needed.

1. If possible, I would like to publish without including test files. Is this feasible, and if so, what steps should I take to exclude test files from the release?

2. After submitting, the Continuous Integration (CI) process fails consistently, displaying the error: "Error: 1:1 error Packages should contain value components, not just types @definitelytyped/no-type-only-packages." Even after reverting the code, the failure persists. Consequently, I added "@definitelytyped/no-type-only-packages" to my .eslintrc.json. If this addition is not allowed, please provide guidance on the necessary modifications.